gcp/main.tf

```diff
@@ -185,7 +213,6 @@ locals {
     db_user         = local.sql_username
     service_account = module.postgres[0].cloudsql_service_account
     bucket          = module.storage.bucket_ref
-    project_id      = var.project_id
```

Comment:
Is this supposed to be removed? I still see it here in https://github.com/bufbuild/terraform-modules-bufstream/blob/e6b50bc3b79d0f543160a937399b7790104360bd/gcp/pg-setup.sh.tpl#L19

Reply:
Yeah - `project_id` is still there. This removal is because it is repeated.

Reply:
oops, nice